.github/workflows/main.yml

```diff
@@ -693,3 +692,96 @@ jobs:
         PR_REF_SHA: ${{ github.event.pull_request.head.sha }}
       if: contains(steps.files.outputs.modified, 'configure.ac') || contains(steps.files.outputs.modified, 'shell/install.sh') || contains(steps.files.outputs.all, 'src_ext') || contains(steps.files.outputs.all, '.github/workflows')
       run: bash -exu .github/scripts/main/hygiene.sh
+ 
+
+  Depends-Linux:
+    runs-on: ubuntu-latest
+    needs: [ Analyse ]
+    strategy:
+      matrix:
+        ocamlv: [ 4.14.2, 5.3.0 ]
+      fail-fast: false
+    env:
+      OPAM_DEPENDS: 1
+    steps:
+    - name: Install bubblewrap
+      run: sudo apt install bubblewrap
+    - name: Disable AppArmor
+      run: echo 0 | sudo tee /proc/sys/kernel/apparmor_restrict_unprivileged_userns
+    - name: Checkout tree
+      uses: actions/checkout@v4
+    - name: src_ext/archives and opam-repository Cache
+      id: archives
+      uses: actions/cache@v4
+      with:
+        path: |
+          src_ext/archives
+          ~/opam-repository
+        key: ${{ needs.Analyse.outputs.archives }}
+        enableCrossOsArchive: true
+    - name: OCaml ${{ matrix.ocamlv }} Cache
+      id: ocaml-cache
+      uses: actions/cache@v4
+      with:
+        path: ~/.cache/ocaml-local/**
+        key: ${{ runner.os }}-ocaml-${{ matrix.ocamlv }}-${{ needs.Analyse.outputs.ocaml-cache }}
+    - name: Create OCaml ${{ matrix.ocamlv }} cache
+      if: steps.ocaml-cache.outputs.cache-hit != 'true'
+      run: bash -exu .github/scripts/main/ocaml-cache.sh ${{ runner.os }} ${{ matrix.ocamlv }}
+    - name: opam bootstrap Cache
+      id: opam-bootstrap
+      uses: actions/cache@v4
+      with:
+        path: |
+          ${{ env.OPAMBSROOT }}/**
+          ~/.cache/opam-local/bin/**
+        key: opamdepends-${{ runner.os }}-${{ env.OPAMBSVERSION }}-${{ matrix.ocamlv }}-${{ env.OPAM_REPO_SHA }}-${{ needs.Analyse.outputs.opam-bs-cache }}
+    - name: Create opam bootstrap cache
+      if: steps.opam-bootstrap.outputs.cache-hit != 'true'
+      run: bash -exu .github/scripts/main/opam-bs-cache.sh
+    - name: Compile
+      env:
+        BASE_REF_SHA: ${{ github.event.pull_request.base.sha }}
+        PR_REF_SHA: ${{ github.event.pull_request.head.sha }}
+        GITHUB_PR_USER: ${{ github.event.pull_request.user.login }}
+      run: bash -exu .github/scripts/main/main.sh x86_64-pc-linux-gnu
+    - name: Comment or update PR with dependency errors
```

Comment:
You need to update ci.ml that generates main.yml, see [error](https://github.com/arozovyk/opam/actions/runs/15879250871/job/44774844704?pr=3#step:7:273)